docs/src/benchcomp-conf.md

```diff
@@ -1,8 +1,41 @@
 # `benchcomp` configuration file
 
 `benchcomp`'s operation is controlled through a YAML file---`benchcomp.yaml` by default or a file passed to the `-c/--config` option.
-This page lists the different visualizations that are available.
+This section documents the top-level schema and the different visualizations
+that are available.
 
+## Configuration file
+
+The schema for `benchcomp.yaml` is shown below:
+
+{{#include ../gen_src/BenchcompYaml.md}}
+
+A minimal example of such a file is:
```

Comment:
It would be great to have a paragraph to explain the workflow of a variant in this configuration.